cmd/aws-application-networking-k8s/main.go

```diff
@@ -151,6 +153,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	grpcRouteReconciler := controllers.NewGrpcRouteReconciler(cloud, mgr.GetClient(), mgr.GetScheme(), mgr.GetEventRecorderFor("grpcroute"), finalizerManager, latticeDataStore)
+
+	if err = grpcRouteReconciler.SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "GRPCRoute")
+		os.Exit(1)
+	}
+
```

Comment:
I did refactoring for other controllers, not http route yet, with single call controller registration. Can you follow that pattern please?

https://github.com/aws/aws-application-networking-k8s/blob/main/cmd/aws-application-networking-k8s/main.go#L142-L145C3

You also need to add logger argument and use it in controller, rather than glog

Reply:
Sure thing, will do next commit!